openlibrary/core/helpers.py

```diff
@@ -33,6 +33,7 @@
 # handy utility to parse ISO date strings
 from infogami.infobase.utils import parse_datetime
 from infogami.utils.view import safeint
+from infogami.utils.i18n import i18n_loadhook
```

Comment:
We don't need to import this anywhere; this function is automagically called every time a request is made (since it's registered here: https://github.com/internetarchive/infogami/blob/8e03b4fb0d844ded794112e90d08da9787ed1ba4/infogami/utils/i18n.py#L200 )
```suggestion
```